.github/workflows/tests_linux_x86_mpi_gpu.yml

```diff
@@ -39,6 +39,8 @@ jobs:
       max-parallel: 1
       matrix:
         mpilib: ["mpich", "openmpi"]
+        cuda_version_maj: ["12"]
```

Comment:
This will likely not be used in a `matrix`, i.e. if we add `"11"` then `cuda_version_min` should not take value `"2"`. So we could maybe just have a single version parameter `cuda_version_maj_min`.

Reply:
While a good idea, we often need separation of major and minor version (in this case between modules, loaded versions etc) that this is just convenient from a QOL perspective. We can migrate it to the JSON mapped style as a follow up, but I think this is fine for now.